doc/navigating.rst

```diff
@@ -158,10 +158,12 @@ that does not change the status of the parent directories.
 Sharing From Your Desktop
 -------------------------
 
-The Nextcloud desktop sync client integrates with your file manager: Finder on
-macOS, Explorer on Windows, and Nautilus on Linux. (Linux users must install 
-the ``Nextcloud-client-nautilus`` plugin.) You can create share links, and share
-with internal Nextcloud users the same way as in your Nextcloud Web interface.
+The Nextcloud desktop sync client integrates with your file manager. Finder on
+macOS and Explorer on Windows. Linux users must install an additional package 
```

Comment:
What about phrasing it like that (because Linux users must only install these packages if they use the distributions package):

Linux users must install an additional package if they installed the client through their distribution, depending on the used file manager. For example, if you are using Ubuntu with the Nautilus file manager, you need to install the package ``nautilus-nextcloud``. If you are using Kubunutu  you are most likely using Dolphin, and need to install ``dolphin-nextcloud``. Other available file manager on Ubuntu are ``nemo-nextcloud`` and ``caja-nextcloud``.

Reply:
Well, your proposal isn't true. Even if they use the provided appimage they will need the extra package for the file manager integration. The appimage won't install such plugins for you.

Reply:
Yes, but for example if you compiled the client yourself, you can also install the file manager plugin directly from the source. You don't need to install the plugins from the distribution. Maybe I make it to complicate here.

Reply:
I see what you mean. Still, "if you compiled the client yourself" is a very strong if. It's a question of who is the target audience of the documentation? I'd argue it's mainly for people who went through https://nextcloud.com/install/#install-clients to install the client. Which means they'd have the appimage and so it should be IMHO the primary supported scenario.

Technical people who build themselves are fringe and probably don't need that documentation in the first place. :-)

Reply:
I would second, that we may expect from users that compile the client themself that they know what they are doing...

Reply:
Alright :) I'll merge then